arcade/types.py

```diff
@@ -408,14 +408,15 @@ def random(
         :param b: Fixed value for blue channel
         :param a: Fixed value for alpha channel
         """
+        rand = random.randint(0, 0xFFFFFFFF)
```

Comment:
This constant is at the top of the same file.
```suggestion
        rand = random.randint(0, MAX_UINT32)
```